### DISCUSSION TITLE
Help field calculator HfcQGIS

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/ods-mkdocs-material#20] [Racconta]
Il bellissimo [**HfcQGIS - Help field calculator**](https://hfcqgis.opendatasicilia.it/) è un esempio.

**Comment by pigreco** [comment]
capitolo 1: `genesi`

Era la primavera del 2018 quando decisi di creare un repository per raccogliere esempi e screenshot sull'uso del field calc di QGIS.

La cosa che mi spinse a fare screenshot e documentare le funzioni fu il fatto, oltre che il numero elevevato di domande in giro per il web su come utilizzare le espressioni di QGIS, che l help in line del field calc non ha nessuna immagine che rappresenti la funzione. La primissima funzione documentata tramite un disegno fu [Azimuth](https://hfcqgis.opendatasicilia.it/gr_funzioni/geometria/geometria_unico/#azimuth)

![image](https://user-images.githubusercontent.com/7631137/138587919-1d703acf-e010-4609-a74b-2843b8a4c5b6.png)

**Comment by pigreco** [comment]
capitolo 2: `prima release in RTD`

Poche settimane dopo la realizzazione del repository in GitHub, vengo contattato da _Andrea Borruso_, _Giovan Battista Vitrano_ e _Ciro Spataro_; creiamo un gruppo telegram e iniziamo a parlare di come creare un sito per rendere più usabile la guida, Ciro consiglia di usare RTD e così nasce la prima release di #HfcQGIS in RTD by ODS (OpenDataSicilia)

![hfc_preloader](https://user-images.githubusercontent.com/7631137/138757072-282fc622-04eb-4eaa-bbec-8622eb120a71.png)
